alchemy-pg_search.gemspec

```
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
   spec.files = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^spec/}) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "alchemy_cms", [">= 5.0", "< 6"]
+  spec.add_runtime_dependency "alchemy_cms", [">= 5.0", "< 7"]
```

Muss angehoben werden

```suggestion
  spec.add_runtime_dependency "alchemy_cms", [">= 6.0", "< 7"]
```